### DISCUSSION TITLE
Hide the pop up message that displays which profile is in use

### DISCUSSION THREAD

**Discussion by Pasteruised** [SaifAqqad/AHK_MicMute#15] [Q&A] [answered]
Hi, is it possible to hide the message that pops up at the bottom of the screen that displays which profile is in use?

**Comment by SaifAqqad** [comment, marked as answer]
Hey, No you can't disable that right now, but I will look into adding this as an option.

Do you think it should be the default behavior when the OSD is disabled?

**Reply by Pasteruised** [reply]
Yeah, that would be perfect.
Keep up the good work man!

**Reply by SaifAqqad** [reply]
Hey, I opted not to change the default behavior of disabling the feedback OSD; instead, I added a new option `SwitchProfileOSD` that you can set to `0` in the config file to disable it. (for now it's not in UI)

You can check it out by downloading the [newest pre-release build](https://github.com/SaifAqqad/AHK_MicMute/releases/tag/latest_commit)

**Reply by Pasteruised** [reply]
Awesome, thank you!
